client/src/pages/LoginPage.js

```diff
@@ -26,6 +27,27 @@ const InputContainer = styled.div`
   animation-fill-mode: forwards;
 `;
 
+const LogInButton = styled(Button)`
+  margin-top: 52px;
+`;
+
+const Container = styled.div`
+  display: flex;
+  align-items: center;
+  font-family: SF Pro Display Regular;
+  font-size: 18px;
+`;
+
+const Info = styled.span`
+  color: ${colors.textInfoDark};
+  margin: 5px;
+`;
+
+const RegisterLink = styled.a`
+  color: ${colors.textActiv};
+  margin: 5px;
+`;
```

Comment:
This component has the same style as the `LoginLink` component. You can reduce code, if you create a generic link component to an outsurced file. Maybe it would also be a better option to set the style for all `<a>` tags in your `GlobalStyles`, so every link in your application will look like this. However, this last option might also affect links on your other pages.